src/extension/wps/wps-core/src/main/java/org/geoserver/wps/xml/WPSInternalXSDParserDelegate.java

```diff
@@ -0,0 +1,30 @@
+package org.geoserver.wps.xml;
+
+import java.lang.reflect.Field;
+
+import org.geotools.xml.Configuration;
+import org.geotools.xml.XSDParserDelegate;
+import org.geotools.xml.impl.ParserHandler;
+import org.springframework.util.ReflectionUtils;
+import org.xml.sax.helpers.NamespaceSupport;
+
+/**
+ * Extends XSDParserDelegate to make package level ParserHandler accesible via protected getter getHandler()
+ * using ReflectionUtils
+ * Adds a NamespaceSupport to default handler
+ * @author GeoSolutions
+ */
+public class WPSInternalXSDParserDelegate extends XSDParserDelegate {
```

Comment:
I would implement this class as a ``private static final`` inner class of ``WPSConfiguration``. No other code will need to access \ use this class.